### PR TITLE
Seaborn support

### PR DESCRIPTION
**Pull request by loiswells97**

### CODE REVIEW

File: src/PyodideWorker.js [hunk 1]
```diff
@@ -249,19 +254,69 @@ const PyodideWorker = () => {
...
+                bytes_io.seek(0)
+                basthon.kernel.display_event({ "display_type": "matplotlib", "content": bytes_io.read() })
+            plt.show = show_chart
+            `);
           return;
         }
       },
+      after: () => {},
+    },
+    seaborn: {
```
Comment (danhalson) [inline src/PyodideWorker.js:274]: I think we're quickly getting to a point here where sticking these package shim things in separate modules might be preferable

File: cypress/e2e/spec-wc-pyodide.cy.js [hunk 2]
```diff
@@ -100,6 +100,29 @@ describe("Running the code with pyodide", () => {
...
   });
 
+  it("runs a simple seaborn program", () => {
+    runCode("import seaborn as sns\ndata = [50, 30, 100]\nsns.displot(data)");
+    cy.wait(12000);
+    cy.get("editor-wc")
+      .shadow()
+      .find(".pyodiderunner")
+      .find("img")
+      .should("be.visible");
```
Comment (danhalson) [inline cypress/e2e/spec-wc-pyodide.cy.js:110]: would a snapshot be a possibility?

File: src/PyodideWorker.js (hunk 1 above)
Comment (loiswells97) [reply src/PyodideWorker.js:274]: Thanks Dan, Ive created another ticket to look into this 👍

File: cypress/e2e/spec-wc-pyodide.cy.js (hunk 2 above)
Comment (loiswells97) [reply cypress/e2e/spec-wc-pyodide.cy.js:110]: Separate ticket created! 👍

**Review by danhalson — APPROVED** [review]
LGTM

**Pull request merged**